imports/client/components/AccountForm.tsx

```diff
@@ -17,13 +17,15 @@ import Form from "react-bootstrap/Form";
 import styled from "styled-components";
 import Flags from "../../Flags";
 import updateProfile from "../../methods/updateProfile";
+import { type LoginOptions } from "../../server/accounts";
```

Comment:
We should pull this into `imports/lib/` if it's going to be used from both client- and server-side code.